packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/camera.dart

```diff
@@ -157,7 +157,7 @@ class CameraUpdate {
   /// Returns a camera update that modifies the camera zoom level by the
   /// specified amount. The optional [focus] is a screen point whose underlying
   /// geographical location should be invariant, if possible, by the movement.
-  static CameraUpdate zoomBy(double amount, [Offset focus]) {
+  static CameraUpdate zoomBy(double amount, [Offset? focus]) {
```

Comment:
Optional positional arguments are generally discouraged. @cyanglaz Should we use the fact that this will be a breaking change PR anyway to make this a named parameter?